portfolio/src/main/java/com/google/sps/servlets/GlobalWarmingDataServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns global warming sentiment data as a JSON object */
+@WebServlet("/global-warming-sentiment-data")
+public class GlobalWarmingDataServlet extends HttpServlet {
+
+  private LinkedHashMap<String, Integer> globalWarmingSentiment = new LinkedHashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+        "/WEB-INF/global-warming-sentiment-data.csv"));
+    
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
```

Comment:
instead of cells choose a naming

tweetAndOpinion / hahah that name doesn't sound good either but choose a better name than cells. We're not sure what cells mean until we read the next lines